### ISSUE TITLE
Thoughts on Security Recolored into Blue.

### ISSUE THREAD

**Issue by C4BRA** [VestaOfOrion/Vesta.Bay#204]
#### Description of issue

**RECOLORING SECURITY TO BLUE IS A BAD IDEA**

Security is a important department that has always been recognized by its signature aggressive red color. Changing the security department's color to blue, and modifying all the uniforms, hardsuits, etc into blue is a bad idea. 

**MAJOR REASON: TWO DEPARTMENTS ALREADY USE BLUE + HUMAN PSYCHOLOGY**

Command already uses a lighter blue for their department
Medical uses a mix of white and light blue for their department.
These two departments already use the color blue, what is the point in changing the security department's color to blue. The color blue is associated to feelings of calmness, non-threatening, while red is associated with aggressiveness, power, and dominance. If security no longer is red, then the color red is not used as a primary color in any department on the Dagon.  

Here is some food for thought:
[https://www.psychologytoday.com/us/blog/mind-games/201502/red-vs-blue-which-should-you-choose](url)

TL:DR: Red brings up dominant behaviors in the person who represents/wears/etc it, while people who view red finds them intimidating or imposing.

I have no idea why people believe using a color that is already used in two departments is a good idea. It removes a huge chunk of uniqueness that is given to security, it removes that certain feeling when dealing with security, etc.

**Comment by gilvian** [comment]
Counter point

![image](https://user-images.githubusercontent.com/13832819/103440106-37588500-4c43-11eb-82e6-2f0ae9ed76c8.png)

Also its just a color, its not really that big of a problem, other servers that have blue sec never got a problem with people mistaking Medical and Security and the blue uniforms look a bit better than red warcriminal vibes

**Comment by BurpleBineapple** [comment]
As already said, other servers have done the same in the past with no real issue. Please keep in mind that GitHub's issues page is not for subjective complaints, just bug reports. Keep stuff like this to the respective discord channels
